### PR TITLE
ft(signup)add image upload on signup or social media profile picture …

### PR DESCRIPTION
**Pull request by Orlayhemmy**
#### What does this PR do?
	•	installed packages
	•	add env file
	•	store social media app secrets and Ids in .env
	•	disable avatar selection on signup
	•	refactor user controller on server-side to es6

#### Description of Task to be completed?
It will replace avatar field with users social media profile picture when they are authenticated
#### How should this be manually tested?

When a user clicks on any of the social media icons, the user will be redirected to their website and will be asked to input their login credentials and submit it. The user will be redirected back to cfh website when the credentials are verified and authenticated.

#### What are the relevant pivotal tracker stories?
Story Id #156882173

### CODE REVIEW

File: public/js/init.js [hunk 1]
```diff
@@ -16,7 +16,14 @@ const checkAuthenticationStatus = (token) => {
...
+  if (window.location.pathname === "/auth/google/callback"
+    || window.location.pathname === "/auth/facebook/callback"
+    || window.location.pathname === "/auth/twitter/callback"
+    || window.location.pathname === "/auth/github/callback"){
+    window.location = '/';
+  }
+  
+};
 
+checkAuthenticationStatus(token);
```
Comment (emmabaye) [inline public/js/init.js]: Nice work. However, there should be a new line at the end of this file

Comment (Orlayhemmy) [reply public/js/init.js]: I have done that..tnx

**Review by emmabaye — COMMENTED** [review]
Great job implementing social authentication.

**Review by amarachukwu-agbo — COMMENTED** [review]
Well done Olayemi, implementing the sign-up image feature

**Pull request merged**